Lang support and version upgrade.

Emetico opened the pull request:
Added language support, based on umbracos langs and upgraded spectrum from version 1.2.0 to 1.8.0. Also added format config option, so rgba can be used.

robertjf commented:
Hey Bo,

Finally got around to publishing the changes in Our and on NuGet – thanks for your hard work 😊

Robert Foster

0418 131 065 / robert@refoster.com.au<mailto:robert@refoster.com.au>

R & E Foster & Associates Pty Ltd

From: Bo Jacobsen <notifications@github.com>
Sent: Friday, 8 December 2017 7:24 AM
To: robertjf/UmbColourPicker <UmbColourPicker@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Subject: [robertjf/UmbColourPicker] Lang support and version upgrade. (#1)


Added language support, based on umbracos langs and upgraded spectrum from version 1.2.0 to 1.8.0. Also added format config option, so rgba can be used.

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/robertjf/UmbColourPicker/pull/1

Commit Summary

  *   Lang support and version upgrade.

File Changes

  *   M README.md<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-0> (5)
  *   M Source/Package/6390AFFD-0056-49A5-9597-AA9B0FAA2FB6/package.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-1> (96)
  *   M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/SpectrumColorPicker.controller.js<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-2> (80)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/da-DK.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-3> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/de-DE.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-4> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/en-GB.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-5> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/en-US.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-6> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/es-ES.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-7> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/fr-FR.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-8> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/he-IL.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-9> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/it-IT.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-10> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/ja-JP.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-11> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/nl-NL.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-12> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/pl-PL.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-13> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/pt-BR.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-14> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/ru-RU.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-15> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/sv-SE.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-16> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/tr-TR.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-17> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/zh-CN.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-18> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/zh-TW.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-19> (11)
  *   A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lib/prevalue/prefferedFormat.html<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-20> (7)
  *   M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.css<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-21> (84)
  *   M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.js<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-22> (2143)
  *   M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/package.manifest<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-23> (10)
  *   M Source/SpectrumColourPicker/SpectrumColourPicker.csproj<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-24> (4)

Patch Links:

  *   https://github.com/robertjf/UmbColourPicker/pull/1.patch
  *   https://github.com/robertjf/UmbColourPicker/pull/1.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/robertjf/UmbColourPicker/pull/1>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAx7tUguYRg9Lm2cIJeKPKlNqv-b80oQks5s-FeKgaJpZM4Q6N0d>.

Emetico commented:
Hi Robert.

Its my pleasure.

Sendt fra min iPhone

Den 22. feb. 2018 kl. 08.47 skrev Robert Foster <notifications@github.com<mailto:notifications@github.com>>:

Hey Bo,

Finally got around to publishing the changes in Our and on NuGet – thanks for your hard work 😊

Robert Foster

0418 131 065 / robert@refoster.com.au<mailto:robert@refoster.com.au><mailto:robert@refoster.com.au>

R & E Foster & Associates Pty Ltd

From: Bo Jacobsen <notifications@github.com<mailto:notifications@github.com>>
Sent: Friday, 8 December 2017 7:24 AM
To: robertjf/UmbColourPicker <UmbColourPicker@noreply.github.com<mailto:UmbColourPicker@noreply.github.com>>
Cc: Subscribed <subscribed@noreply.github.com<mailto:subscribed@noreply.github.com>>
Subject: [robertjf/UmbColourPicker] Lang support and version upgrade. (#1)


Added language support, based on umbracos langs and upgraded spectrum from version 1.2.0 to 1.8.0. Also added format config option, so rgba can be used.

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/robertjf/UmbColourPicker/pull/1

Commit Summary

* Lang support and version upgrade.

File Changes

* M README.md<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-0> (5)
* M Source/Package/6390AFFD-0056-49A5-9597-AA9B0FAA2FB6/package.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-1> (96)
* M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/SpectrumColorPicker.controller.js<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-2> (80)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/da-DK.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-3> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/de-DE.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-4> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/en-GB.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-5> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/en-US.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-6> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/es-ES.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-7> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/fr-FR.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-8> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/he-IL.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-9> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/it-IT.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-10> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/ja-JP.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-11> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/nl-NL.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-12> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/pl-PL.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-13> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/pt-BR.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-14> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/ru-RU.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-15> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/sv-SE.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-16> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/tr-TR.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-17> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/zh-CN.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-18> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lang/zh-TW.xml<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-19> (11)
* A Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lib/prevalue/prefferedFormat.html<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-20> (7)
* M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.css<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-21> (84)
* M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.js<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-22> (2143)
* M Source/SpectrumColourPicker/App_Plugins/SpectrumColorPicker/package.manifest<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-23> (10)
* M Source/SpectrumColourPicker/SpectrumColourPicker.csproj<https://github.com/robertjf/UmbColourPicker/pull/1/files#diff-24> (4)

Patch Links:

* https://github.com/robertjf/UmbColourPicker/pull/1.patch
* https://github.com/robertjf/UmbColourPicker/pull/1.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/robertjf/UmbColourPicker/pull/1>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AAx7tUguYRg9Lm2cIJeKPKlNqv-b80oQks5s-FeKgaJpZM4Q6N0d>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/robertjf/UmbColourPicker/pull/1#issuecomment-367539850>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AJpU8uZqIMVCDEMymeVhFA8KDbywxeWaks5tXMcWgaJpZM4Q6N0d>.
